README.md

```diff
@@ -137,6 +139,7 @@ Following inputs can be used as `step.with` keys
 | `git_push_gpgsign`**¹**               | Bool    | Sign all pushes automatically. (default `false`) |
 | `git_committer_name`**¹**             | String  | Set commit author's name (defaults to the name associated with the GPG key) |
 | `git_committer_email`**¹**            | String  | Set commit author's email (defaults to the email address associated with the GPG key) |
+| `workdir`**¹**                        | String  | Working directory (below repository root) |
```

Comment:
Remove `**¹**`

Reply:
Taking a look at how that got added.

Reply:
I copied the line above. Dohhh... It's fixed now.